README.md

```diff
@@ -71,7 +71,7 @@ After:
 ```
 
 ```js
-export default class Component.extend({
+export default Component.extend({
```

Comment:
Hmm, let's go the other direction (if you don't mind); Let's do this:

```js
export default class extends Component {
```

What do you think @skaterdav85?

Reply:
@rwjblue sounds good. I'll update.

Reply:
@rwjblue done